toolbox

```diff
@@ -964,7 +964,12 @@ remove_containers()
             ret_val=$(echo "$ids" \
                       | (
                             while read -r id; do
-                                if ! $prefix_sudo podman rm $force_option "$id" >/dev/null 2>&3; then
+                                $prefix_sudo podman rm $force_option "$id" >/dev/null 2>&3
+                                ret_code=$?
```

Comment:
Don't we also need to handle exit code 1?

Even though we have previously seen that the container does exist, it's possible that it got deleted by a simultaneously executing toolbox script or podman command or something else.